test/functional/fixtures/regression/gh-423/test.js

```diff
@@ -0,0 +1,25 @@
+describe('[Regression](GH-423)', function () {
+    it('Should raise click event except in Firefox if target element appends child after mousedown', function () {
+        return runTests('testcafe-fixtures/index.test.js', 'Raise click if target appends child', { skip: ['firefox', 'firefox-osx'] });
```

Comment:
you can write just `skip: ['firefox']` because of [this `indexOf` check](https://github.com/DevExpress/testcafe/blob/master/test/functional/setup.js#L142)